Cartfile

```diff
@@ -1,3 +1,3 @@
 github "Alamofire/Alamofire" ~> 4.0.0
-github "ReactiveCocoa/ReactiveSwift" "1.0.0-alpha.2"
-github "ReactiveX/RxSwift" "3.0.0-rc.1"
+github "ReactiveCocoa/ReactiveSwift" "1.0.0-alpha.3"
+github "ReactiveX/RxSwift" "3.0.0"
```

Comment:
I think this should either be `~> 3.0.0` or `== 3.0.0`
https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement